livekit-android-sdk/src/main/java/io/livekit/android/room/participant/LocalParticipant.kt

```diff
@@ -419,6 +419,30 @@ internal constructor(
         }
     }
 
+    /**
+     * Control who can subscribe to LocalParticipant's published tracks.
+     *
+     * By default, all participants can subscribe. This allows fine-grained control over
+     * who is able to subscribe at a participant and track level.
+     *
+     * Note: if access is given at a track-level (i.e. [allParticipantsAllowed] and
+     * [ParticipantTrackPermission.allTracksAllowed] is false), any newer published tracks
```

Comment:
:nit: `are false`

Reply:
maybe even better to add the word `both` after `i.e.`, something like `i.e. both [allParticipantsAllowed]...` to make it very explicit.